dom0-updates/qubes-dom0-update

```diff
@@ -128,15 +124,18 @@ if [ "$CHECK_ONLY" == "1" ]; then
 fi
 
 # Redirect operations on templates to qvm-template command
-if find_regex_in_args '^qubes-template-' "${PKGS[@]}"; then
+if has_template "${PKGS[@]}"; then
     if [[ ${#PKGS[@]} -ne 1 ]]; then
         echo "ERROR: Specify only one package to reinstall template"
         exit 2
     fi
-    ONEPKG="$(echo "${PKGS[0]}" | sed -r 's/-[0-9]+(\.[0-9-]+)+(\.noarch)*$//')" # Remove version suffix
-    TEMPLATE=${ONEPKG#qubes-template-} # Remove prefix
+    if [[ "${PKGS[0]}" =~ ^qubes-template-(.*)(-[0-9]+)(\.[0-9-]+)+(\.[0-9a-zA-Z_]+)?$ ]]; then
+        TEMPLATE=${BASH_REMATCH[1]}
+    else
+        TEMPLATE=${PKGS[0]#qubes-template-}
+    fi
     echo "Redirecting to 'qvm-template ${YUM_ACTION=install} ${QVMTEMPLATE_OPTS[*]} $TEMPLATE'"
```

Comment:
If you change it below, change YUM_ACTION here too. It is set already when adding to PKGS.